### ISSUE TITLE
Add ability to set different search engine for each collection

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#3163]

**Comment by maxxxxxdlp** [comment]
This is possible in a hacky way on `xml-editor` branch:
Add a URL like this as your search engine: `<specify7Domain>/specify/command/switch-collection/<collectionId>?nextUrl=%2Fexpress-search%2F%3F%3D<searchQueryGoesHere>`

the search would switch collection first, and then redirect to the search page

Though, having a cleaner way would be better

**Comment by maxxxxxdlp** [comment]
> Not yet, unfortunately. It can only be configured for the entire database at the moment, but this could be added in the future!

Is express search actually querying the entire database? I though it was scoped to logged in collection

**Comment by grantfitzsimmons** [comment]
No, it is not searching the database, but it is scoped to the database, based on your current logged in collection
